osh/hub/settings.py

```diff
@@ -186,6 +189,18 @@
 except ImportError:
     pass
 
+
+def _get_secret(name):
+    try:
+        with open(os.path.join(SECRETS_DIR, name)) as f:
+            return f.read().strip()
+    except OSError:
```

Comment:
@kdudka Should we also silently skip permission related errors or just missing files?

Reply:
Assigning `None` (at least in case of missing files) sounds fine to me.  The hub will continue to work until the secret is actually needed.  Not everybody uses Bugzilla or Jira.  But we should probably return `None` explicitly in this case.

Reply:
But `None` is returned even now, though not explicitly.  (Any Python function/method without explicit `return` statement returns `None`.)

Reply:
oh you're right, either way it's clearer to write it out I guess.